data/recipes/grr_artifact_ts.json

```diff
@@ -45,7 +45,7 @@
         ["--approvers", "Emails for GRR approval request.", null],
         ["--sketch_id", "Timesketch sketch to which the timeline should be added.", null],
         ["--wait_for_timelines", "Whether to wait for Timesketch to finish processing all timelines.", true],
-        ["--analyzers", "Timesketch analysers to run", null],
+        ["--analyzers", "Timesketch analyzers to run", null],
```

Comment:
`analysers` is not a typo, it's UK English spelling.

Reply:
You're right. Just notation is different, not a typo!
However, my intention to amend that was "unification".
The option was labeled `--analyzers`, but the corresponding description was `analysers`.
Please tell me your thoughts on revising with these intentions! I'll reflect it right away.

Reply:
Ahh excellent point, we should keep it the same.